src/backend/joanie/core/models/accounts.py

```diff
@@ -114,6 +124,19 @@ class Meta:
                 fields=["owner"],
                 name="unique_main_address_per_user",
             ),
+            models.UniqueConstraint(
+                condition=models.Q(is_main=True),
+                fields=["organization"],
+                name="unique_main_address_per_organization",
+            ),
```

Comment:
I feel something is wrong with those two `UniqueConstraint`.

I'll test that all is working fine by writing a test which would try to create two UserAddress or OrganizationAddress. I suspect that an error will be raised as two main addresses will have an empty relation (owner or organization = `None`)

Reply:
I've just made a test, and it seems to work fine. Let's discuss about it.

Reply:
Let's discuss about this tomorrow. I wrote 2 tests which highlights some error.